src/search-headless.ts

```diff
@@ -437,7 +440,8 @@ export default class SearchHeadless {
       context,
       referrerPageUrl,
       locationRadius,
-      additionalHttpHeaders: this.additionalHttpHeaders
+      additionalHttpHeaders: this.additionalHttpHeaders,
+      queryId
```

Comment:
nit: Similar to Emily's comment above, if possible I'd prefer if we could have additionalHttpHeaders as the last parameter, and thus should add queryId before it. Basically have all the very specific params first and then the additional http headers.

Reply:
done